api/src/play_asset_pack_manager.gd

```diff
@@ -132,7 +142,9 @@ func _initialize_plugin() -> Object:
 # _asset_pack_to_request_map.
 # -----------------------------------------------------------------------------
 func _remove_request_reference_from_map(pack_name : String):
-	_asset_pack_to_request_map.erase(pack_name)
+	var erase_success = _asset_pack_to_request_map.erase(pack_name)
```

Comment:
Another way to suppress this warning as well as the unused variable warning is to replace these three lines using:
```
			var _erase_success = _asset_pack_to_request_map.erase(pack_name)
```
Is this a better solution comparing to the one we are currently using?

Reply:
Like my comment in the request tracker: This is a good option, but only if you actually want to consider this an error. If the error case could be hit during normal operation it's better to ignore the return value.

Reply:
Just want to confirm, is this a case that the developer should be aware of, or would this be hit during normal operation and they don't really care if it happens?

Reply:
Developers should not be aware of that since and is impossible to hit that during normal operation.